ament_cmake_export_dependencies/cmake/ament_cmake_export_dependencies-extras.cmake.in

```diff
@@ -36,11 +36,11 @@ if(NOT _exported_dependencies STREQUAL "")
 
           get_target_property(_imported_configurations ${_target} IMPORTED_CONFIGURATIONS)
           if(_imported_configurations)
-            get_target_property(_imported_implib ${_target} IMPORTED_IMPLIB_${_imported_configurations})
+            get_target_property(_imported_implib ${_target} IMPORTED_IMPLIB_$<CONFIG>)
             if(_imported_implib)
               list(APPEND _libraries "${_imported_implib}")
             else()
-              get_target_property(_imported_location ${_target} IMPORTED_LOCATION_${_imported_configurations})
+              get_target_property(_imported_location ${_target} IMPORTED_LOCATION_$<CONFIG>)
```

Comment:
it seems that generator expressions can't be used in `get_target_property` either, so `$<CONFIG>` is not being solved correctly :confused: